Packs/Slack/Integrations/SlackV3/SlackV3.py

```diff
@@ -1485,7 +1490,35 @@ async def listen(client: SocketModeClient, req: SocketModeRequest):
             entitlement_string = json.loads(entitlement_json)
             entitlement_reply = json.loads(entitlement_json).get("reply", "Thank you for your reply.")
             action_text = actions[0].get('text').get('text')
-            answer_question(action_text, entitlement_string, user.get('profile', {}).get('email'))
+            incident_id = answer_question(action_text, entitlement_string, user.get('profile', {}).get('email'))
+            if state and DEMISTO_API_KEY:
```

Comment:
can it be that we will have a state and the DEMISTO_API_KEY is missing because of wrong config?
if so, shouldn't we log this explicitly?

Reply:
State would show up if they are sending out any blocks from the app which reports back a state. I think a log line here wouldn't be a bad idea purely from a "Hey you didn't add an API key" sense.